Program.cs

```diff
@@ -1,410 +1,427 @@
 ﻿using System;
-using VisualCode.Entities;
-using System.Collections.Generic;
 using System.Linq;
+using System.Collections.Generic;
+using VisualCode.Entities;
 
 namespace VisualCode
 {
     class Program
     {
+        enum enumMenuMain : int {Maintenance = 1, RegistrySales = 2, Reports = 3};
+        enum enumMenuMaintenance : int {Customers = 1, Products = 2, Sellers = 3, MenuPrincipal = 4};
+        enum enumMenuReport : int {Sales = 1, CustomersAge = 2, CustomersBirthdayMonth = 3, ProductsStock = 4};
+
+        static List<Customers> customersList = new List<Customers>();
+        static List<Products> productsList = new List<Products>();
+        static List<Sellers> sellersList = new List<Sellers>();
+        static List<Invoices> invoicesList = new List<Invoices>();
+        static List<InvoiceDetails> invoiceDetailsList = new List<InvoiceDetails>();
+
         static void Main(string[] args)
         {
-            List<Customers> customersList = new List<Customers>();
-            List<Product> productList = new List<Product>();
-            List<Seller> sellerList = new List<Seller>();
-            List<Invoice> invoiceList = new List<Invoice>();
-            List<InvoiceDetail> invoiceDetailList = new List<InvoiceDetail>();
-
             LoadDataDefault();
 
-            MenuPrincipal();
-            ProcesoPrincipal();
+            MenuMain();
+            ProcessMain();
+        }
 
-            //Console.WriteLine($"escribiste {linea}");
+        private static void ProcessMain()
+        {
+            if (!int.TryParse(Console.ReadLine(), out int option))

```

Comment:
Infact you can use Enum.TryParse and use it for the switch without other cast more